src/components/speakers/SpeakerDirectory.tsx

```
@@ -0,0 +1,205 @@
+    type GuestSpeaker,
+    type SpeakerHistory,
+} from "@/lib/speakerDirectory";
+
+export function SpeakerDirectory() {
+    const [searchQuery, setSearchQuery] = useState("");
+    const [searchResults, setSearchResults] = useState<GuestSpeaker[]>([]);
+    const [isSearching, setIsSearching] = useState(false);
+    const [selectedSpeaker, setSelectedSpeaker] = useState<SpeakerHistory | null>(null);
+    const [isLoadingHistory, setIsLoadingHistory] = useState(false);
+
+    useEffect(() => {
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)